nomad-mixin/mixin.libsonnet

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Looks like you based your traefik branch off of the nomad branch, so these commits are included.

That's fine, but make sure you merge the nomad PR before this one, just for tidyness sake.